1.12/Dockerfile

```diff
@@ -6,10 +6,12 @@ ENV SERVICE_ROOT /service
 ENV SERVICE_USER service
```

Comment:
Should we move these envs down/declare them right before they are used in the build process?

Reply:
These are static, so keeping them at the top is probably best so that this layer doesn't change